app/config/config.yml

```diff
@@ -28,6 +28,11 @@ framework:
     session:
         # handler_id set to null will use default session handler from php.ini
         handler_id:  ~
+        cookie_lifetime: 604800
+        # http://symfony.com/doc/current/components/http_foundation/session_configuration.html#configuring-garbage-collection
+        gc_maxlifetime: 604800
+        gc_probability: 1
+        gc_divisor: 100
```

Comment:
Are you sure you need to change the `gc_*` values? I expect the defaults work fine with this longer cookie lifetime